build.gradle

```diff
@@ -539,30 +560,123 @@ publishing {
+            pom {
+                name = 'HiveMQ Community Edition Embedded'
+                description = 'HiveMQ Community Edition Embedded'
+                url = 'http://www.hivemq.com/'
+                licenses {
+                    license {
+                        name = 'The Apache License, Version 2.0'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    }
+                }
+                developers {
+                    developer {
+                        id = 'cschaebe'
+                        name = 'Christoph Schaebel'
+                        email = 'christoph.schaebel@hivemq.com'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
                     }
                     developer {
                         id = 'lbrandl'
                         name = 'Lukas Brandl'
-                        email = 'lukas.brandl@dc-square.de'
-                        organization = 'dc-square GmbH'
-                        organizationUrl = 'https://www.dc-square.de'
+                        email = 'lukas.brandl@hivemq.com'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
                     }
                     developer {
                         id = 'flimpoeck'
                         name = 'Florian Limpoeck'
                         email = 'florian.limpoeck@dc-square.de'
-                        organization = 'dc-square GmbH'
-                        organizationUrl = 'https://www.dc-square.de'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
+                    }
+                    developer {
+                        id = 'sauroter'
+                        name = 'Georg Held'
+                        email = 'georg.held@hivemq.com'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
+                    }
+
+                    developer {
+                        id = 'SgtSilvio'
+                        name = 'Silvio Giebl'
+                        email = 'silvio.giebl@hivemq.com'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
                     }
                 }
                 scm {
                     connection = 'scm:git:git@github.com:hivemq/hivemq-community-edition.git'
                     developerConnection = 'scm:git:git@github.com:hivemq/hivemq-community-edition.git'
                     url = 'https://github.com/hivemq/hivemq-community-edition/tree/master'
                 }
+
+            }
+
+            pom.withXml {
+                asNode().dependencies.dependency.each { dep ->
+                    if (dep.artifactId.last().value().last() in ["logback-classic", "slf4j-log4j12"]) {
```

Comment:
we should use the java-library plugin and use the right dependency scopes
hacking the pom manually is error-prone

Reply:
I am not sure if this is solved. Please take a look again.

Reply:
Actually we could use optional features with their own dependency configurations. Then the dependencies would be listed as optional in the pom

Reply:
But I do not think this should delay this ticket.